idb/postgres/postgres_migrations_test.go

```diff
@@ -24,11 +24,44 @@ func TestConvertAccountDataIncrementsMigrationNumber(t *testing.T) {
 	err := db.setMigrationState(nil, &migrationState)
 	require.NoError(t, err)
 
-	err = convertAccountData(&db, &migrationState, nil)
+	err = convertAccountData(&db, &migrationState)
 	require.NoError(t, err)
 
 	migrationState, err = db.getMigrationState(context.Background(), nil)
 	require.NoError(t, err)
 
 	assert.Equal(t, types.MigrationState{NextMigration: 6}, migrationState)
 }
+
+func TestConvertBigIntType(t *testing.T) {
```

Comment:
Not sure if this is necessary, but should we add a test case that asserts the existing values aren't changed by the type conversion?

i.e. insert round 1,000,000 convert the type and assert that the round is still 1,000,000?

Reply:
good point. I think this can be covered by running the block validator.